.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!--
+If you are reporting a new issue, make sure that we do not have any duplicates
+already open. If there is a duplicate, please close your issue and add a comment
+to the existing issue.
+
+If you suspect your issue is a bug, please edit your issue description to
+include the information shown below.
+
+For more information about reporting issues, see
+https://github.com/openwhisk/openwhisk/blob/master/CONTRIBUTING.md#raising-issues
+
+
+Use the commands below to provide key information from your environment:
+You do NOT have to include this information if this is a feature request
+-->
+
+**Environment details (Operating System, ...):**
```

Comment:
use # for section headers everywhere instead of * fill some details

#### Environment details
e.g. local deployment, vagrant, native ubuntu, Mac OS, Bluemix,
e.g. version of docker, vagrant, ubuntu, ...